ui/app/pages/settings/contact-list-tab/view-contact/view-contact.component.js

```diff
@@ -55,11 +56,9 @@ export default class ViewContact extends PureComponent {
               >
                 { quadSplit(checkSummedAddress) }
               </div>
-              <img
-                className="address-book__view-contact__group__static-address--copy-icon"
-                onClick={() => copyToClipboard(checkSummedAddress)}
-                src="/images/copy-to-clipboard.svg"
-              />
+              <button className="address-book__view-contact__group__static-address--copy-icon" onClick={() => copyToClipboard(checkSummedAddress)}>
```

Comment:
Nit: could you put each of these props on a new line? It requires horizontal scrolling when viewing on GitHub at the moment.